### ISSUE TITLE
Synthetic hurricane model bug

### ISSUE THREAD

**Issue by pwolfram** [MPAS-Dev/MPAS-Model#625]
Use of the `testing_and_setup/compass/ocean/hurricane_wind_pressure/main.py` file produces a faulty last time step for `testing_and_setup/compass/ocean/hurricane/synthetic_hurricanes/SANDY_017771.json` synthetic hurricane using input file `hurricane_inputs.txt`:
```
SANDY_017771.json                                                                                       
2▸        ▸ ! grid flag 1 = regular grid (need further inputs), 2 = nc file (need file name)
mesh.nc
1013                    ! mbar, ambient pressure
1.                      ! holland B parameter
```
                                                                                                        
where `output.nc` has the faulty field that is not define for the last time step, resulting in spurious large number in MPAS-O:
```python
ipython
In [7]: import xarray as xr                                                                                                                                                                                                                                                                                                  

In [8]: ds = xr.open_dataset('out.nc')                                                                                                                                                                                                                                                                       

In [9]: ds.windSpeedU.isel(Time=-1)                                                                                                                                                                                                                                                                                          
Out[9]: 
<xarray.DataArray 'windSpeedU' (nCells: 15404)>
array([9.96921e+36, 9.96921e+36, 9.96921e+36, ..., 9.96921e+36, 9.96921e+36,
       9.96921e+36])
Dimensions without coordinates: nCells

In [10]: ds.windSpeedU.isel(Time=-2)                                                                                                                                                                                                                                                                                         
Out[10]: 
<xarray.DataArray 'windSpeedU' (nCells: 15404)>
array([-3.588561,  1.057031,  0.461513, ..., -6.030833,  6.671099,  6.515482])
Dimensions without coordinates: nCells
In [11]: ds.xtime[-2:]                                                                                                                                                                                                                                                                                                       
Out[11]: 
<xarray.DataArray 'xtime' (Time: 2)>
array([b'2012-01-12_23:00:00                                             ',
       b'2012-01-13_00:00:00                                             '],
      dtype='|S64')
Dimensions without coordinates: Time
```

**Comment by pwolfram** [comment]
Bug found in https://github.com/MPAS-Dev/MPAS-Model/pull/624

**Comment by sbrus89** [comment]
FIxed by #626
